assets/css/styles.css

```diff
@@ -511,7 +522,7 @@ h1.title {
 }
 
 #site-footer a:hover {
-  color: #fff;
+  color: #0092cf;
```

Comment:
Please add any colors that are repeated across pages to the `:root` at the top of the `styles.css` file. You can then use the color with the syntax:
`color: var(--color-name);`